lib/fuse_lowlevel.c

```diff
@@ -446,7 +446,7 @@ int fuse_reply_entry(fuse_req_t req, const struct fuse_entry_param *e)
 int fuse_reply_create(fuse_req_t req, const struct fuse_entry_param *e,
 		      const struct fuse_file_info *f)
 {
-	char buf[sizeof(struct fuse_entry_out) + sizeof(struct fuse_open_out)];
+	uint64_t buf[(sizeof(struct fuse_entry_out) + sizeof(struct fuse_open_out) + 7) / 8];
```

Comment:
I would prefer to avoid magic numbers and I guess an ROUND_UP macro would do the job? From my uring branch, where I need it to allocate buffers:

```
#define ROUND_UP(val, round_to) \
	(((val) + (round_to - 1)) & ~(round_to - 1))
``

Reply:
Done